schema.sql

```diff
@@ -0,0 +1,12 @@
+/* This is the database schema for the vet_clinic database which defines the structure of the entire database, including the 'animals' table. */
+CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals (
+  id INT,
```

Comment:
- [ ] (optional) You could make the `id` autoincremented

Reply:
Thanks I have changed it to automatically increments its value for each new row inserted into the table.😁